distro/pkg/rpm/netopeer2.spec

```diff
@@ -65,13 +65,12 @@ a single established NETCONF session.
 %cmake -DCMAKE_BUILD_TYPE=RELWITHDEBINFO \
        -DSYSREPO_SETUP=OFF \
        -DPIDFILE_PREFIX=/run \
-       -DSERVER_DIR=%{_libdir}/netopeer2-server
+       -DSERVER_DIR=%{_var}/lib/netopeer2-server
 %cmake_build
 
 %install
 %cmake_install
 install -D -p -m 0644 %{SOURCE2} %{buildroot}%{_unitdir}/netopeer2-server.service
-mkdir -p -m=700 %{buildroot}%{_libdir}/netopeer2-server
```

Comment:
Don't you need to `mkdir %{_sharedstatedir}/netopeer2-server` somewhere, and add it to the `%files` section as well? Or will the server create it at runtime?

Reply:
The server did create the directory at runtime in my experience, which matches the code:

https://github.com/CESNET/netopeer2/blob/1e639fe18a70b06111ad9c3ea261f27794f25f0e/src/netconf_confirmed_commit.c#L207-L217

As `/var` exists for services to write to at runtime, I thought this behavior was okay.

But I don't mind creating it and adding it to files either. Let me know what you prefer.

Reply:
I checked the "docker.io" package on Ubuntu, and it does not include `/var/lib/docker`, but the daemon creates it at runtime.
I don't know docker can be considered as a standard to adhere to, though..

Reply:
Since this is a `.spec` file, that means RPM, so I suggest to look at RPM-based distros. Fedora is the most common one (at least regarding various distro-specific rules).

Reply:
You are absolutely right, RPM based Distros seem to follow a different convention, I found several packages that do include their /var/lib directories in the RPM. Commit updated.